### ISSUE TITLE
Add an axis argument to the cylindrical_radius, cylindrical_theta

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1452]
Jose would like at a minimum that the documentation mention that it using the z axis for the axis of rotation. He would also like to be able to specify the axis.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. As such, not all 
information was able to be captured in the transition. Below is
a complete record of the original redmine ticket.

Ticket number: 3105
Status: Resolved
Project: VisIt
Tracker: Feature
Priority: Normal
Subject: Add an axis argument to the cylindrical_radius, cylindrical_theta
Assigned to: Mark Miller
Category: -
Target version: 2.13.3
Author: Eric Brugger
Start: 05/15/2018
Due date: 
% Done: 0%
Estimated time: 
Created: 05/15/2018 04:09 pm
Updated: 10/17/2018 08:35 pm
Likelihood: 
Severity: 
Found in version: 2.12.3
Impact: 3 - Medium
Expected Use: 3 - Occasional
OS: All
Support Group: Any
Description:
 Jose would like at a minimum that the documentation mention that it using the z axis for the axis of rotation. He would also like to be able to specify the axis.

Comments:
Basically, Jose wants here to define a variable on the mesh which represents distance from some arbitrary line (e.g. axis) going through the mesh which is possible though non-trivial by construction from other expressions. The documentation has been updated. I need to close loop with Jose regarding ability to use combination of transform operator, defer expression operator, etc. to permit computation of a cylindrical_radius from an arbitrary line.
